Streetcode/Streetcode.BLL/MediatR/Streetcode/RelatedFigure/GetByStreetcodeId/GetRelatedFiguresByStreetcodeIdHandler.cs

```diff
@@ -62,7 +62,7 @@ public async Task<Result<IEnumerable<RelatedFigureDTO>>> Handle(GetRelatedFigure
         return Result.Ok(_mapper.Map<IEnumerable<RelatedFigureDTO>>(relatedFigures));
     }
 
-    private IQueryable<int> GetRelatedFigureIdsByStreetcodeId(int StreetcodeId)
+    private IQueryable<int>? GetRelatedFigureIdsByStreetcodeId(int StreetcodeId)
```

Comment:
return `Enumerable.Empty<int>().AsQueryable()` in the catch block and remove nullable in the return type

The `if (relatedFigures is null)` statement above will be replaced with `if (!relatedFigureIds.Any())` after my PR